### ISSUE TITLE
Simulation Archives do not carry over units

### ISSUE THREAD

**Issue by rabagoi** [hannorein/rebound#297]
When accessing a simulation from a Simulation Archive, using `sim.units` will display the units as 

`{'length': None, 'mass': None, 'time': None}`

instead of the units set in the original simulation.

This doesn't seem to have any impact on how the simulations run; any simulations created from the archive appear to run with the "correct" units.

**Comment by hannorein** [comment]
@dtamayo Do you have an idea on how to fix this? It's a bit tricky because units only exist in python. One could add three fields corresponding to units to the simulation struct in c. But it adds all kinds of potential issues, i.e. these fields are not used in C, python needs to check those field every time a simulation in loaded from a binary file. The latter might be hard because the loading from a binary file is done in C. We can brain storm in person... maybe there is a smart way to implement this.

**Comment by dtamayo** [comment]
I agree. Might be good to think about in general for fields that only exist in Python. Would love to talk about it in person
